packages/manager/apps/pci-private-network/src/hooks/useColumns/useColumns.tsx

```diff
@@ -0,0 +1,235 @@
+                        slot="start"
+                      >
+                        {t(
+                          'pci_projects_project_network_private_subnet_create',
+                        )}
+                      </OsdsText>
+                    </OsdsButton>
+                  ),
+                },
+                {
+                  id: 2,
+                  content: (
+                    <OsdsButton
+                      size={ODS_BUTTON_SIZE.sm}
+                      variant={ODS_BUTTON_VARIANT.ghost}
+                      color={ODS_THEME_COLOR_INTENT.primary}
+                      disabled
+                    >
+                      <OsdsText
+                        size={ODS_THEME_TYPOGRAPHY_SIZE._500}
+                        level={ODS_TEXT_LEVEL.button}
+                        color={ODS_TEXT_COLOR_INTENT.primary}
+                        slot="start"
+                      >
+                        {t('pci_projects_project_network_private_subnet_show')}
+                      </OsdsText>
+                    </OsdsButton>
+                  ),
+                },
+                {
+                  id: 3,
+                  content: (
+                    <OsdsButton
+                      size={ODS_BUTTON_SIZE.sm}
+                      variant={ODS_BUTTON_VARIANT.ghost}
+                      color={ODS_THEME_COLOR_INTENT.primary}
+                      disabled
+                    >
+                      <OsdsText
+                        size={ODS_THEME_TYPOGRAPHY_SIZE._500}
+                        level={ODS_TEXT_LEVEL.button}
+                        color={ODS_TEXT_COLOR_INTENT.primary}
+                        slot="start"
+                      >
+                        {t('pci_projects_project_network_private_delete')}
+                      </OsdsText>
+                    </OsdsButton>
+                  ),
+                },
+              ]}
+            />
+          </DataGridTextCell>
+        ),
+        label: t('pci_projects_project_network_private_action'),
+      },
+    ],
+    [t],
+  );
+}
+
+type LZPrivateNetworkColumn = TGroupedSubnet & { isPending: boolean };
+
+export function usePrivateNetworkLZColumns() {
+  const { t } = useTranslation(['listing', 'common']);
+  const navigate = useNavigate();
+
+  return useMemo(
+    () => [
+      {
+        id: 'name',
+        cell: ({ name }: LZPrivateNetworkColumn) => (
+          <DataGridTextCell>{name}</DataGridTextCell>
+        ),
+        label: t('pci_projects_project_network_private_name'),
+      },
+      {
+        id: 'region',
+        cell: ({ region }: LZPrivateNetworkColumn) => (
+          <DataGridTextCell>{region}</DataGridTextCell>
+        ),
+        label: t('pci_projects_project_network_private_region'),
+      },
+      {
+        id: 'cidr',
+        cell: ({ cidr, isPending }: LZPrivateNetworkColumn) => (
+          <SkeletonWrapper isPending={isPending}>
```

Comment:
Using skeleton wrapper like this can cause error on property access, it is fine here but could lead to errors in other usages